netstandard/pkg/NETStandard.Library.pkgproj

```diff
@@ -31,6 +33,10 @@
     <File Include="..\tools\targets\netstandard1.7\$(TargetsFileName)">
       <TargetPath>build/netstandard1.7/$(Id).targets</TargetPath>
     </File>
+
+    <Dependency Include="_._">
```

Comment:
Should we put this directly under netstandard? i.e. for all netstandard versions.